src/main/resources/application.yaml

```diff
@@ -49,8 +49,10 @@ quarkus:
           policy: authenticated
 
     limits:
-      # Let's lower maximum HTTP request size from 10 megs to 1
-      max-body-size: 1M
+      # Let's limit low-level maximum HTTP request size to 10 megs: stricter limits
+      # are applied at the JSON API level. Low-level limits may result in EPIPE/413 errors
+      # whereas at higher level we can use regular JSON API error responses
+      max-body-size: 10M
```

Comment:
what's the default here, do we even need to explicitly define it?

Reply:
I think it is 10 megs. We do not absolutely have to define it, although it may make sense for documentation purposes.